### PR TITLE
Remove unnecessary input field on add another pattern

### PR DESCRIPTION
**Pull request by ian-leggett**
## Problem
As part of #1803 I am removing an input field that we don't need in the add another pattern as it just creates unnecessary space on the page. I have spoken with other teams and they agree we should remove it.
Screenshot 2019-03-25 at 09 44 18

## Solution
Remove the input field.
Screenshot 2019-03-25 at 09 46 45

### CODE REVIEW

File: assets/javascripts/vue/typeahead.vue [hunk 1]
```diff
@@ -162,6 +162,7 @@
         this.setPlaceHolder = this.placeholder
       },
       getLabelFromValue: function (value, model) {
```
Comment (yozzo) [inline assets/javascripts/vue/typeahead.vue]: in ES6 you can assign a default value where you declare the argument `function (value = '', model, model={})`

File: assets/javascripts/vue/typeahead.vue [hunk 2]
```diff
@@ -162,6 +162,7 @@
...
+        if(!value){ return }
```
Comment (zakdit) [inline assets/javascripts/vue/typeahead.vue]: Not anything big but in single line if statements you don't need the parenthesis, you can just do  `if(!value) return `

Comment (zakdit) [reply assets/javascripts/vue/typeahead.vue]: Also consider doing `if(!value || !model) return` or is `model` always being passed through?

File: assets/javascripts/vue/typeahead.vue (hunk 1 above)
Comment (ian-leggett) [reply assets/javascripts/vue/typeahead.vue]: Good point let me update this

File: assets/javascripts/vue/typeahead.vue (hunk 2 above)
Comment (ian-leggett) [reply assets/javascripts/vue/typeahead.vue]: Thanks Zak looking at this now

**Pull request merged**